CONTRIBUTORS/IIIT Gwalior/Second Year/Divya Agrawal/info.md

```diff
@@ -0,0 +1 @@
+Hello everyone, Myself Divya Agrawal and I'm a second year undergraduate from IIIT Gwalior.
```

Comment:
Add your portfolio link here in the info.md file
And make sure you have done issue #1 earlier, cause you have not mentioned #1, so you won't get the points, in the PR. Tag me once in the GitHub & Discord when you complete the requested changes.